manual/speculative_execution/README.md

```diff
@@ -121,14 +118,22 @@ explicitly depend on it:
 </dependency>
 ```
 
-Then create an instance of [PerHostPercentileTracker][phpt] that will collect
-latency statistics for your `Cluster`:
+Then create a [PercentileTracker] that will collect latency histograms for your `Cluster`. Two
+implementations are provided out of the box:
+
+* [PerHostPercentileTracker]: maintains a histogram per host. This means queries to a host will
+  only be compared against previous queries to the same host.
+* [ClusterWidePercentileTracker]: maintains a single histogram for the whole cluster. This means
```

Comment:
small nit: Since we recommend ClusterWidePercentileTracker we should list it first.